test/api/events/list.test.ts

```diff
@@ -85,6 +85,30 @@ test("GET /events/list", async (t: ExecutionContext) => {
   t.true(events_list_request.data.events.length === 1)
   t.is(events_list_request.data.events[0]?.device_id, device_id)
 
+  // Test 200 response (device_ids)
+  events_list_request = await axios.get("/events/list", {
+    params: {
+      device_ids: [device_id],
+      since: date_before_event_happened,
+    },
+    headers: {
+      Authorization: `Bearer ${seed.ws2.cst}`,
+    },
+  })
+  t.is(events_list_request.data.events.length, 0)
+
+  // Test 200 response (empty device_ids)
+  events_list_request = await axios.get("/events/list", {
```

Comment:
@andrii-balitskyi Note that we should try to break up things into individual tests like here: https://github.com/seamapi/fake-seam-connect/pull/117/files#diff-8a407f687fb4396e3aa1629fd81c6e1bfa3175eaad5a817051cc8eeaacb3823cR70

I know there is some setup involved to get the events in this case, but I think we can do that in a `test.beforeEach` hook and do `t.context.date_before_event_happened = date_before_event_happened` etc.